### ISSUE TITLE
RuAun Gardens

### ISSUE THREAD

**Issue by m241dan** [DarkstarProject/xiNavmeshes#1]
I don't know if you are taking issues, but in RuAun Gardens, Dolls that are in the wall don't seem to aggro properly(like they can't see me because Im a taru) and Despot paths very awkwardly around the little ponds.

**Comment by bendangelo** [comment]
Yes i'm accepting issues. How do dolls aggro? Mobs take line of sight into account now.

How does Despot path around ponds? And how does that compare to retail?

**Comment by m241dan** [comment]
The dolls in the hole should aggro when you stand right infront of them, but they don't. Almost as if the lip that each "wall hole" is blocking it. They will aggro if you move off to the side of the lips that jut out, but even then usually only to one side of those lips and not both.

As for Despot, I'm sure all mobs that path will have this problem, but they can't get past the first tree between the pond and the VuLegannon build entrances. Also, when he paths around the ponds, occasionally, he actually falls down in them and moves around inside them which he shouldn't be able to. 

I will grab some POS for the Despot tree.

**Comment by m241dan** [comment]
Despot around the lake: there seems to be a wall that runs perpendicular with the pond/building wall at 120 -24 -428.

The other side doesn't have this wall but the gap between the pond and the tree at -123 -24 -420 and 123 -24 -420 needs to be a little wider, they should be able to move through it no problem and occasionally it will force the mob down into the pond(shouldn't be able to get into the pond at all) where it will then walk around.

A "hole doll" example: The Groundskeeper should aggro at 10 -43 -259 and if I take a few steps backs as well but it doesn't. It will only aggro if I stand super close and awkwardly off to the side at 14 -43 -264.

Hope this is way more informative than my first posts!
